packages/fhir-team-management/src/components/AddEditOrganization/index.tsx

```diff
@@ -74,7 +74,8 @@ export const AddEditOrganization = (props: AddEditOrganizationProps) => {
 
   if (
     (!organization.isIdle && organization.isLoading) ||
-    (!practitioners.isIdle && practitioners.isLoading)
+    (!practitioners.isIdle && practitioners.isLoading) ||
```

Comment:
is the `isIdle` state check necessary if we are checking `isLoading` on the same resource?

Reply:
I think its because the query can be `not enabled` and thus idle, so its not running, its has not loaded its not loading,